src/main/java/Task.java

```diff
@@ -0,0 +1,37 @@
+public class Task {
+
+    /** Description of task */
+    protected String description;
+    /** Boolean indicating if task is done */
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public boolean isDone() {
+        return isDone;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public void setDone(boolean done) {
+        this.isDone = done;
+    }
+
+//    @Override
```

Comment:
would be better to remove comments that are not used in the actual code when u push your code to github